docs/Extract-Client-Data.md

```diff
@@ -15,26 +15,26 @@ Next press Configue and then Generate.
 ### Now that you have the source compiled, you can proceed to the next step.
 
 3. Go to your AzerothCore build directory (E.g. AzerothCore\build\bin\Release) and copy the following files to your World of Warcraft directory (E.g. D:\World of Warcraft 3.3.5a (no install))
-* **ace.dll**
 * **mapextractor.exe**
 * **mmaps_generator.exe**
 * **vmap4assembler.exe**
 * **vmap4extractor.exe**
-* **[extract_vmaps.bat](https://pastebin.com/KKMurZ4D)**
+* **[extract_vmaps.bat](https://pastebin.com/KKMurZ4D)** : You need to create a .txt file, write on it with the linked text, save it and then rename the file as **extract_vmaps.bat**
```

Comment:
uhm, actually it looks like a notepad window-based solution. You can safely create an extract_vmap.bat file and edit it with vscode. Ain't?

Reply:
You can edit it with any kind of text editor. Notpad++, or even the default ones if you use "Open With" so it is not necessary no

Reply:
But isnt the extract tools included with the build of the soruce?

Reply:
not, that one, tha's why there's a link : )